src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -398,6 +406,8 @@ protected function do_import( Tribe__Events__Importer__File_Importer $importer )
 
 		$new_offset = $importer->import_complete() ? -1 : $importer->get_last_completed_row();
 		update_option( 'tribe_events_importer_offset', $new_offset );
+		global $wpdb;
+		$wpdb->query("SELECT RELEASE_LOCK('tribe_ea_csv_import')");
```

Comment:
* :warning: Usage of a direct database call is discouraged.
* :warning: Direct database call without caching detected. Consider using wp_cache_get() / wp_cache_set() or wp_cache_delete().
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: c2b51084630bc7d92efa8c98da9090b2</sub>